core/src/main/java/io/questdb/std/DirectLongList.java

```diff
@@ -45,7 +45,7 @@ public class DirectLongList implements Mutable, Closeable, Reopenable {
     public DirectLongList(long capacity, int memoryTag) {
         this.memoryTag = memoryTag;
         this.capacity = (capacity * Long.BYTES);
-        this.address = Unsafe.malloc(this.capacity, memoryTag);
+        this.address = Unsafe.calloc(this.capacity, memoryTag);
```

Comment:
I think it's better to keep `malloc` rather than `calloc` here. For instance, we allocate a rather big list for async offload's filtered row ids, but if the filter returns only a few rows, most of the memory is left uninitialized. So, this change may lead to unwanted side effects.

Reply:
fixed in https://github.com/questdb/questdb/pull/3399/commits/59d4a04ba2e67e5e6cb50069d3f4c44777e3b112